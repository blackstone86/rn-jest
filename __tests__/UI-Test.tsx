import React from 'react';
import {View} from 'react-native';
import {create} from 'react-test-renderer';

const ViewBox = () => <View style={{backgroundColor: 'white'}} />;

const tree = create(<ViewBox />);

test('snapshot[component]: ViewBox', () => {
  expect(tree).toMatchSnapshot();
});
