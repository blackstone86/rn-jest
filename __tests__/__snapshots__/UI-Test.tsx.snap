// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot[component]: ViewBox 1`] = `
<View
  style={
    Object {
      "backgroundColor": "white",
    }
  }
/>
`;
